minimal/glsp-server/.vscode/tasks.json

```
@@ -0,0 +1,32 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Build Minimal GLSP Server",
+      "type": "shell",
+      "group": {
+        "kind": "build",
+        "isDefault": true
+      },
+      "presentation": {
+        "reveal": "always",
+        "panel": "new"
+      },
+      "command": "yarn ",
```

This is fixed with https://github.com/eclipse-glsp/glsp-examples/pull/89, but for the sake of completeness:
```suggestion
      "command": "mvn clean verify",
```